sql-plugin/src/main/340/scala/org/apache/spark/sql/rapids/shims/RapidsErrorUtils.scala

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
all of the files for 3.4.0 should go into a 340+ directory with the assumption that they will work for other versions like 3.4.1.